packages/xo-web/src/xo-app/logs/backup-ng/log-alert-body.js

```diff
@@ -301,11 +305,7 @@ const TaskLi = ({ className, task, ...props }) => {
   ) {
     return null
   }
-  return (
-    <li className={className}>
-      <Component task={task} {...props} />
-    </li>
-  )
+  return <Component task={task} {...props} />
```

Comment:
Why this change? 
So, the `TaskLi` isn't correct, right?

Reply:
- Q1: To let the component decide if it want to be displayed or not. See https://github.com/vatesfr/xen-orchestra/blob/d84bc42a8e945c6cb8f5b60a107b54cf4bc2aa81/packages/xo-web/src/xo-app/logs/backup-ng/log-alert-body.js#L254-L257
- Q2: It's correct because the components are an item of the list.